packages/web/src/containers/NavBarContainer.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```

Comment:
Acho que como o Rogério disse, não sera necessárioum container para NavBar, logo, assumo que estes estados serão  colocados no container pai que a navbar estiver, mas acho que por enquanto podemos deixar assim para ter a lógica de comportamento separada do componente.

Reply:
Achei q era só do progress bar, mas se for do nav bar tbm eu tiro.